b2sdk/sync/policy.py

```diff
@@ -65,10 +87,23 @@ def _should_transfer(self):
             return True
         else:
             # Both exist.  Transfer only if the two are different.
```

Comment:
It's more complicated than "only if the two are different".  Not sure how much detail should be in this comment.

Reply:
this comment is very old and not part of this PR changes, but I agree, it can be misleading and needs to go.